### DISCUSSION TITLE
"Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."

### DISCUSSION THREAD

**Discussion by oyontalas** [community/community#43664] [Copilot Conversations]
### Select Topic Area

General

### Body

I'm a student, and I'm certified as a student by github. I saw that github student certified users can use copilot for free. But I got a notification today that the use of the copilot extension has ended. How should I do?
![image](https://user-images.githubusercontent.com/82634912/211228174-6f8dce50-d5d5-4b8a-9d8c-fee37458e800.png)

**Comment by jongwooo** [comment]
@oyontalas [Others](https://github.com/community/community/discussions/19050) are also raising similar issues. I hope it is resolved quickly.

**Comment by otkrickey** [comment]
Solved: #43634

**Comment by danvufs** [comment]
Mine has not solved! ( Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended.
You are eligible to use GitHub Copilot for free.)

**Comment by GrinZero** [comment]
I also encountered this problem

**Comment by euwbah** [comment]
Just encountered

**Comment by AlphaWhiskyLou** [comment]
Just encountered the same problem.

**Comment by dvalley56** [comment]
Oh yes me too. Yesterday I was getting rate limit exceeded for plan EDUCATION something like that, but sometime it worked. Whatever it is, hope it gets fixed soon.

**Comment by BIT-Xu** [comment]
same

**Comment by fahad0samara** [comment]
same

**Comment by Brajham** [comment]
Ya se soluciono!!!! Automaticamente pero tengo una duda Por que me Aparece Copilot de esta manera me limita o bueno tengo que realizar varios saltos de linea para que se ejecute el copilot
![copilot](https://user-images.githubusercontent.com/98179164/211978970-e1e83eab-cb02-4bfd-b0e4-43c50ce6fd5e.jpg)
es muy molesto :( 
tengo mi cuenta verificada Education o esto solo me pasa a mi?

**Comment by Aholicknight** [comment]
Having the same problem as well and I just re-verified, hopefully this will fix it.

**Reply by Brajham** [reply]
bro como lo volviste a verificar? me podrias dar detalles por favor

**Reply by kevocde** [reply]
Looks like a platform problem bro, just wait ...

**Comment by asg0451** [comment]
same :( :(

**Comment by Kaybe2020** [comment]
same (T_T)

**Comment by paulwababu** [comment]
same error, i hope they arent frustrating us into paying!

**Reply by SergioRgVz** [reply]
Same error here, but I am not even allowed to pay it because it does not give me that option in the webpage...

**Comment by PeronGH** [comment]
Me too :(

**Reply by PeronGH** [reply]
Mine is solved automatically. Maybe they already knew this bug and fixed it

**Comment by GWDx** [comment]
same problem

**Comment by Siddhu2502** [comment]
same problem here why am i getting that error and not able to access copilot someone give me how to fix it

**Comment by krishnerkar** [comment]
<img width="797" alt="image" src="https://user-images.githubusercontent.com/49949733/213365261-a9ceacd4-4b03-4ef3-bf4e-d0915d6b43df.png">

i'm eligible to use it but i cant :(

**Reply by scottybollinger** [reply]
> <img alt="image" width="797" src="https://user-images.githubusercontent.com/49949733/213365261-a9ceacd4-4b03-4ef3-bf4e-d0915d6b43df.png">
> 
> i'm eligible to use it but i cant :(

Same. Anyone know how to solve?

**Reply by JohnDominicJasmin** [reply]
i fixed mine by reinstalling github copilot plugin

**Comment by JohnDominicJasmin** [comment]
Try to reinstall your github copilot plugin

**Comment by kolan4ick** [comment]
same

**Comment by Overlytic** [comment]
I am getting the same error. Any ideas what to try?
